<?php

// This file is automatically generated by the latest tag in the git repository when a new commit is pushed
$version = @file_get_contents("/home/multiqc/version.txt");
if($version === FALSE) $version = '?';
else preg_replace('/[^\d\.]/', '', $version);

// Parse the list of modules from the docs folder
require_once("../Spyc.php");
$md = file_get_contents('../multiqc/docs/README.md');
$md_parts = explode('---', $md, 3);
$modules = [];
if(count($md_parts) == 3){
  $pages = spyc_load($md_parts[1]);
  if(isset($pages['MultiQC Modules'])){
    foreach($pages['MultiQC Modules'] as $sect_name => $sect){
      foreach($sect as $fn){
        $mod_md = file_get_contents('../multiqc/docs/'.trim($fn));
        $mod_md_parts = explode('---', $mod_md, 3);
        if(count($mod_md_parts) == 3){
          $yaml = spyc_load($mod_md_parts[1]);
          $yaml['docs_url'] = 'docs/#'.strtolower(str_replace(' ', '-', $yaml['Name']));
          $yaml['section'] = $sect_name;
          $modules[] = $yaml;
        }
      }
    }
  }
}


?><!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <title>MultiQC</title>
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="apple-touch-icon" sizes="57x57" href="/apple-icon-57x57.png">
    <link rel="apple-touch-icon" sizes="60x60" href="/apple-icon-60x60.png">
    <link rel="apple-touch-icon" sizes="72x72" href="/apple-icon-72x72.png">
    <link rel="apple-touch-icon" sizes="76x76" href="/apple-icon-76x76.png">
    <link rel="apple-touch-icon" sizes="114x114" href="/apple-icon-114x114.png">
    <link rel="apple-touch-icon" sizes="120x120" href="/apple-icon-120x120.png">
    <link rel="apple-touch-icon" sizes="144x144" href="/apple-icon-144x144.png">
    <link rel="apple-touch-icon" sizes="152x152" href="/apple-icon-152x152.png">
    <link rel="apple-touch-icon" sizes="180x180" href="/apple-icon-180x180.png">
    <link rel="icon" type="image/png" sizes="192x192"  href="/android-icon-192x192.png">
    <link rel="manifest" href="/manifest.json">
    <meta name="msapplication-TileColor" content="#ffffff">
    <meta name="msapplication-TileImage" content="/ms-icon-144x144.png">
    <meta name="theme-color" content="#ffffff">

    <link href="css/bootstrap.min.css" rel="stylesheet">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="fonts/font-mfizz.css">
    <link rel="stylesheet" href="css/styles.css">

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>

  <body id="mqc_homepage">

    <div class="header">

      <!-- Static navbar -->
      <nav class="navbar navbar-default navbar-inverse navbar-static-top">
        <div class="container">
          <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
              <span class="sr-only">Toggle navigation</span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand visible-xs" href="#">MultiQC</a>
          </div>
          <div id="navbar" class="navbar-collapse collapse">
            <ul class="nav navbar-nav navbar-right">
              <p class="navbar-text hidden-xs hidden-sm"><a href="https://github.com/ewels/MultiQC/releases" class="navbar-link">Current version: <?php echo $version; ?></a></p>
              <li class="active"><a href="#">Home</a></li>
              <li><a href="docs/">Docs</a></li>
              <li><a href="plugins.html">Plugins</a></li>
              <li><a href="logos/">Logo</a></li>
              <li class="dropdown">
                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Example Reports <span class="caret"></span></a>
                <ul class="dropdown-menu">
                  <li class="dropdown-header">Analysis Types</li>
                  <li><a href="examples/rna-seq/multiqc_report.html">RNA-Seq</a></li>
                  <li><a href="examples/wgs/multiqc_report.html">Whole Genome</a></li>
                  <li><a href="examples/bs-seq/multiqc_report.html">Bisulfite</a></li>
                  <li><a href="examples/hi-c/multiqc_report.html">Hi-C</a></li>
                  <li class="dropdown-header">MultiQC Customisation</li>
                  <li><a href="examples/ngi-rna/Test_NGI_Project_multiqc_report.html">MultiQC_NGI Plugin</a></li>
                </ul>
              </li>
            </ul>
          </div><!--/.nav-collapse -->
        </div>
      </nav>

      <div class="container">
        <div class="row">
          <div class="col-md-8">
            <h1>
              <object type="image/svg+xml" title="MultiQC" data="images/MultiQC_logo.svg">
                <img src="images/MultiQC_logo.png">
              </object>
              <span class="__dimensions_badge_embed__ hidden-xs" data-id="pub.1010049881" data-style="large_rectangle" data-legend="never" style="float:right; margin-top: 100px;"></span>
            </h1>
            <p class="lead">Aggregate results from bioinformatics analyses across many samples into a single report</p>
            <p>MultiQC searches a given directory for analysis logs and compiles a HTML report.
              It's a general use tool, perfect for summarising the output from numerous bioinformatics tools.</p>

            <div class="row videos-row">
              <div class="col-md-6">
                <div class="embed-responsive embed-responsive-16by9 homepage-header-video hidden-xs hidden-sm">
                  <iframe id="multiqc-video" class="embed-responsive-item" src="https://www.youtube.com/embed/BbScv9TcaMg?rel=0&amp;showinfo=0" allowfullscreen></iframe>
                </div>
              </div>
              <div class="col-md-6" id="multiqc-videos">
                <div class="btn-group btn-group-xs video-chooser-lang-btn-group" role="group" aria-label="...">
                  <a type="button" class="btn btn-dark" href="#videos-english">English ðŸ‡¬ðŸ‡§</a>
                  <a type="button" class="btn btn-dark" href="#videos-spanish">EspaÃ±ol ðŸ‡ªðŸ‡¸</a>
                </div>
                <div class="video-lang-buttons" id="videos-english" aria-expanded="true">
                  <ul class="list-switch left video-chooser">
                    <li><a href="https://youtu.be/BbScv9TcaMg" data-src="https://www.youtube.com/embed/BbScv9TcaMg?rel=0&amp;showinfo=0" class="active"><span class="hidden-lg hidden-md label label-default">Video</span> Introduction to MultiQC <em>(1:19)</em></a></li>
                    <li><a href="https://youtu.be/Gg5neIPuiVo" data-src="https://www.youtube.com/embed/Gg5neIPuiVo?rel=0&amp;showinfo=0"><span class="hidden-lg hidden-md label label-default">Video</span> Installing MultiQC <em>(4:33)</em></a></li>
                    <li><a href="https://youtu.be/cktKbESOOno" data-src="https://www.youtube.com/embed/cktKbESOOno?rel=0&amp;showinfo=0"><span class="hidden-lg hidden-md label label-default">Video</span> Running MultiQC <em>(5:21)</em></a></li>
                    <li><a href="https://youtu.be/qPbIlO_KWN0" data-src="https://www.youtube.com/embed/qPbIlO_KWN0?rel=0&amp;showinfo=0"><span class="hidden-lg hidden-md label label-default">Video</span> Using MultiQC Reports <em>(6:06)</em></a></li>
                  </ul>
                </div>
                <div class="video-lang-buttons" id="videos-spanish" aria-expanded="false" style="display:none;">
                  <ul class="list-switch left video-chooser">
                    <li><a href="https://youtu.be/BbScv9TcaMg" data-src="https://www.youtube.com/embed/WX_qVhJ32g0?rel=0&amp;showinfo=0" class="active"><span class="hidden-lg hidden-md label label-default">Video</span> IntroducciÃ³n a MultiQC <em>(1:43)</em></a></li>
                    <li><a href="https://youtu.be/Gg5neIPuiVo" data-src="https://www.youtube.com/embed/E_NJCxwXAGQ?rel=0&amp;showinfo=0"><span class="hidden-lg hidden-md label label-default">Video</span> InstalaciÃ³n de MultiQC <em>(4:19)</em></a></li>
                    <li><a href="https://youtu.be/cktKbESOOno" data-src="https://www.youtube.com/embed/0MfW7Y8Ocak?rel=0&amp;showinfo=0"><span class="hidden-lg hidden-md label label-default">Video</span> Â¿CÃ³mo ejecutar MultiQC? <em>(5:53)</em></a></li>
                  </ul>
                </div>
              </div>
            </div>

          </div>
          <div class="col-md-4" id="header-buttons">
            <a class="btn btn-block btn-lg btn-primary" href="https://www.github.com/ewels/MultiQC">
              <i class="fa fa-github fa-lg"></i> GitHub
            </a>
            <a class="btn btn-block btn-lg btn-primary" href="https://pypi.python.org/pypi/multiqc">
              <i class="icon-python fa-lg"></i>  Python Package Index
            </a>
            <a class="btn btn-block btn-lg btn-primary" href="docs">
              <i class="fa fa-book fa-lg"></i> Documentation
            </a>
            <a href="#supported-tools" class="btn btn-block btn-lg btn-primary" data-toggle="modal" data-target="#supported-tools">
              <i class="fa fa-bar-chart"></i> <?php echo count($modules); ?> supported tools
            </a>
            <a href="#publication" class="btn btn-block btn-lg btn-primary" data-toggle="modal" data-target="#publication">
              <i class="fa fa-file-text"></i>  Publication / Citation
            </a>
            <button type="button" class="btn btn-block btn-lg btn-primary" id="gitter_btn">
              <img height="18" src="images/gitter.svg"> Get help on Gitter
            </button>
            <a class="visible-xs visible-sm btn btn-block btn-lg btn-primary" href="examples/rna-seq/multiqc_report.html">
              <i class="glyphicon glyphicon-search"></i> Example Report
            </a>
            <div class="panel panel-primary" id="quick_install">
              <div class="panel-heading">
                <i class="fa fa-terminal"></i> Quick Install <br>
              </div>
              <div class="panel-body">
                <pre id="pip"><code >pip install multiqc    <span class="comment"># Install</span>
multiqc .              <span class="comment"># Run</span></code></pre>
<pre id="conda" style="display:none;"><code>conda install -c bioconda -c conda-forge multiqc
multiqc .</code></pre>
<pre id="manual" style="display:none;"><code>git clone https://github.com/ewels/MultiQC.git
python setup.py install
multiqc .</code></pre>
                <div class="btn-group btn-group-justified btn-group-xs install-switcher" role="group">
                  <a href="#pip" class="btn active">pip</a>
                  <a href="#conda" class="btn">conda</a>
                  <a href="#manual" class="btn">manual</a>
                </div>
              </div>
            </div>
            <p class="sub-panel">Need a little more help? <a href="docs">See the full installation instructions</a>.</small></p>

          </div>
        </div>

      </div> <!-- end of header -->
    </div>

    <!-- Demo Reports -->
    <div class="container hidden-xs hidden-sm">

      <h2>Example Reports</h2>
      <div class="row">
        <div class="col-md-4">
          <ul class="list-switch right demo-chooser">
            <li><a href="examples/rna-seq/multiqc_report.html" data-target="#demo-rna-description"  class="active">RNA-Seq</a></li>
            <li><a href="examples/wgs/multiqc_report.html" data-target="#demo-wgs-description">Whole-Genome Sequencing</a></li>
            <li><a href="examples/bs-seq/multiqc_report.html" data-target="#demo-bsseq-description">Methylation (Bisulfite)</a></li>
            <li><a href="examples/hi-c/multiqc_report.html" data-target="#demo-hic-description">Genome Structure (Hi-C)</a></li>
            <li><a href="examples/ngi-rna/Test_NGI_Project_multiqc_report.html" data-target="#demo-ngi-rna-description">Customised with MultiQC_NGI Plugin</a></li>
          </ul>
        </div>
        <div class="col-md-8 demo-descriptions">
          <div id="demo-rna-description">
            <p>This report was generated using logs from an analysis accidentally run on ChIP-Seq data from
              the <em>BI Human Reference Epigenome Mapping Project: ChIP-Seq in human subject</em> dataset
               (<a href="http://trace.ncbi.nlm.nih.gov/Traces/sra/?study=SRP001534" target="_blank">SRP001534</a>).</p>
            <p>Initial QC was done using <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastQC</a>,
              followed by trimming with <a href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">TrimGalore!</a>
              (a wrapper around <a href="https://github.com/marcelm/cutadapt" target="_blank">cutadapt</a>).
            Reads were aligned using <a href="https://github.com/alexdobin/STAR" target="_blank">STAR</a> and overlaps
            counted with <a href="http://bioinf.wehi.edu.au/featureCounts/" target="_blank">featureCounts</a>.
            <!-- The <a href="http://rseqc.sourceforge.net/" target="_blank">RSeQC</a> package was used to check
            various quality control aspects of the data. -->
          </p>
            <div class="btn-group pull-right" role="group">
              <a href="examples/rna-seq/multiqc_report.zip" class="btn btn-default">
                <span class="fa fa-file-text-o" aria-hidden="true"></span> Download report
              </a>
              <a href="examples/rna-seq/data.zip" class="btn btn-default">
                <span class="fa fa-files-o" aria-hidden="true"></span> Download logs
              </a>
            </div>
            <p>You can download this report and / or the logs used to generate it, to try running MultiQC yourself.</p>
          </div>
          <div id="demo-wgs-description" style="display: none;">
            <p>The data from this report comes from an analysis of HapMap trio samples, run by the
            <a href="https://www.scilifelab.se/platforms/ngi/" target="_blank">National Genomics Infrastructre</a>
            (NGI) at SciLifeLab, Sweden. Initial quality control was done using <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastQC</a>
            and <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/" target="_blank">FastQ Screen</a>.
            Reads were processed with <a href="https://www.broadinstitute.org/gatk/" taret="_blank">GATK</a> and the aligned reads analysed using
            <a href="http://broadinstitute.github.io/picard/" target="_blank">Picard</a>. Downstream QC was done using
            <a href="http://qualimap.bioinfo.cipf.es/" target="_blank">Qualimap BamQC</a> and
            <a href="http://snpeff.sourceforge.net/" target="_blank">SnpEff</a>.</p>
            <div class="btn-group pull-right" role="group">
              <a href="examples/wgs/multiqc_report.zip" class="btn btn-default">
                <span class="fa fa-file-text-o" aria-hidden="true"></span> Download report
              </a>
              <a href="examples/wgs/data.zip" class="btn btn-default">
                <span class="fa fa-files-o" aria-hidden="true"></span> Download logs
              </a>
            </div>
            <p>You can download this report and / or the logs used to generate it, to try running MultiQC yourself.
              Note that the example report has some user-specific config settings, seen in the
              <code><a href="examples/wgs/multiqc_config.yaml" target="_blank">multiqc_config.yaml</a></code> file.</p>
          </div>
          <div id="demo-bsseq-description" style="display: none;">
            <p>The example methylation report is based on analysis of data from the GEO NCBI project
            <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE47966" target="_blank">GSE47966</a>,
            from the 2013 Lister <em>et. al.</em> paper <em>Global epigenomic reconfiguration during mammalian brain development</em>.</p>
            <p>Raw data was run through <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastQC</a>
            and trimmed using <a href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">Trim Galore!</a>
            (a wrapper around <a href="https://github.com/marcelm/cutadapt" target="_blank">Cutadapt</a>). Reads were
            aligned, deduplicated and cytosine methylation statuses called using
            <a href="http://www.bioinformatics.babraham.ac.uk/projects/bismark/" target="_blank">Bismark</a>.</p>
          </p>
            <div class="btn-group pull-right" role="group">
              <a href="examples/bs-seq/multiqc_report.zip" class="btn btn-default">
                <span class="fa fa-file-text-o" aria-hidden="true"></span> Download report
              </a>
              <a href="examples/bs-seq/data.zip" class="btn btn-default">
                <span class="fa fa-files-o" aria-hidden="true"></span> Download logs
              </a>
            </div>
            <p>You can download this report and / or the logs used to generate it, to try running MultiQC yourself.</p>
          </div>
          <div id="demo-hic-description" style="display: none;">
            <p>This Hi-C MultiQC report was generated using Hi-C data from <a href="mailto:louise.harewood@cruk.cam.ac.uk">Louise Harewood</a> at
              <a href="http://www.cambridgecancer.org.uk/" target="_blank">CRUK Cambridge Institute</a>.
              Reads were run through <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">FastQC</a>
              and then processed using <a href="http://www.bioinformatics.babraham.ac.uk/projects/hicup/" target="_blank">HiCUP</a> (Hi-C User Pipeline).</p>
            <div class="btn-group pull-right" role="group">
              <a href="examples/hi-c/multiqc_report.zip" class="btn btn-default">
                <span class="fa fa-file-text-o" aria-hidden="true"></span> Download report
              </a>
              <a href="examples/hi-c/data.zip" class="btn btn-default">
                <span class="fa fa-files-o" aria-hidden="true"></span> Download logs
              </a>
            </div>
            <p>You can download this report and / or the logs used to generate it, to try running MultiQC yourself.
              Note that the example report has some user-specific config settings, seen in the
              <code><a href="examples/hi-c/multiqc_config.yaml" target="_blank">multiqc_config.yaml</a></code> file.</p>
          </div>
          <div id="demo-ngi-rna-description" style="display: none;">
            <p>This MultiQC report was generated in combination with the <a href="https://github.com/ewels/MultiQC_NGI" target="_blank">MultiQC_NGI</a>
              plugin. This plugin adds a few new functionalities to MultiQC which are specific to the
              <a href="https://www.scilifelab.se/platforms/ngi/" target="_blank">SciLifeLab National Genomics Infrastruture</a>.
              A new report template (<code>ngi</code>) styles reports with our logo. A new module (<code>NGI-RNAseq</code>) adds
              sample similarity plots generated from custom code in our <a href="https://github.com/SciLifeLab/NGI-RNAseq" target="_blank">RNA pipeline</a>.
              It also loads intersting data fields from our LIMS (eg. RIN score) and puts these into the report.
              Finally, it saves the parsed biofinformatics summary results back in the LIMS for multi-project meta analyses.</p>
            <div class="btn-group pull-right" role="group">
              <a href="examples/ngi-rna/multiqc_report.zip" class="btn btn-default">
                <span class="fa fa-file-text-o" aria-hidden="true"></span> Download report
              </a>
              <a href="examples/ngi-rna/data.zip" class="btn btn-default">
                <span class="fa fa-files-o" aria-hidden="true"></span> Download logs
              </a>
            </div>
            <p>You can download this report and / or the logs used to generate it, to try running MultiQC yourself.
              The <a href="https://github.com/ewels/MultiQC_NGI" target="_blank">MultiQC_NGI</a> package must be installed.
              Note that the example report has some user-specific config settings, seen in the
              <code><a href="examples/ngi-rna/multiqc_config.yaml" target="_blank">multiqc_config.yaml</a></code> file.
              It can also be run with the <code>--test-db</code> parameter, using the
              <a href="examples/ngi-rna/ngi_db_data.json">example data provided</a>.</p>
          </div>
        </div>
      </div>

      <div id="iframe_browser" class="hidden-xs hidden-sm">
        <div id="iframe_browser_header">
          <div id="iframe_browser_buttons"><span></span><span></span><span></span></div>
          <span id="iframe_browser_title">MultiQC Example Reports</span>
          <ul id="iframe_browser_tabs">
            <li><a href="examples/ngi-rna/Test_NGI_Project_multiqc_report.html">MultiQC_NGI</a></li>
            <li><a href="examples/hi-c/multiqc_report.html">Hi-C</a></li>
            <li><a href="examples/bs-seq/multiqc_report.html">Bisulfite Seq</a></li>
            <li><a href="examples/wgs/multiqc_report.html">Whole-Genome Seq</a></li>
            <li class="active"><a href="examples/rna-seq/multiqc_report.html">RNA-Seq</a></li>
          </ul>
        </div>
        <iframe src="examples/rna-seq/multiqc_report.html"></iframe>
      </div>

    </div>

    <div class="container"><div class="content_block">
      <p class="content-lead">Ever spent ages collecting reports and wading through log file output?<br>
          Here's the answer to your frustrations...</p>

            <section>
                <div class="row">
          <div class="col-md-8">
            <h3>Visualise statistics from your pipeline</h3>
                    <p>MultiQC collects numerical stats from each module at the top the report, so that you can track how your data behaves as it proceeds through your analysis.</p>
            <a class="btn btn-lg btn-default" href="examples/rna-seq/multiqc_report.html">View Example Report</a>
          </div>
          <div class="col-md-4 text-center">
            <img class="img-circle img-thumbnail" src="images/general_stats.png" alt="General Statistics" />
          </div>
        </div>
      </section>

            <section>
                <div class="row">
          <div class="col-md-8">
            <h3>Plot all of your samples together</h3>
                    <p>Visualizing your samples together allows detailed comparison, not possible by scanning one report after another.</p>
            <div class="btn-group">
              <button type="button" class="btn btn-lg btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                Example Reports <span class="caret"></span>
              </button>
              <ul class="dropdown-menu">
                <li class="dropdown-header">Analysis Types</li>
                <li><a href="examples/rna-seq/multiqc_report.html">RNA-Seq</a></li>
                <li><a href="examples/wgs/multiqc_report.html">Whole Genome</a></li>
                <li><a href="examples/bs-seq/multiqc_report.html">Bisulfite</a></li>
                <li><a href="examples/hi-c/multiqc_report.html">Hi-C</a></li>
              </ul>
            </div>
          </div>
          <div class="col-md-4 text-center">
            <img class="img-circle img-thumbnail" src="images/overlay_plot.png" alt="FastQC GC Overlay Plot" />
          </div>
        </div>
      </section>

      <section>
        <div class="row">
          <div class="col-md-8">
            <h3>Supports your favourite tools</h3>
              <p>MultiQC comes supports many common bioinformatics tools out of the box. If you're missing something,
              just create an issue on GitHub to request it - if you have an example log file it's usually pretty fast.</p>
            <a href="#supported-tools" class="btn btn-default btn-lg" data-toggle="modal" data-target="#supported-tools">
              View available Modules
            </a>
          </div>
          <div class="col-md-4 text-center">
            <div class="img-circle img-thumbnail">
              <div class="tool-list">
                <ul class="list-inline">
                  <?php foreach($modules as $mod){
                    echo '<li>'.$mod['Name'].'</li>';
                  } ?>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </section>

      <section>
        <div class="row">
          <div class="col-md-8">
            <h3>Extensible and documented</h3>
              <p>Want to use this to do something fancy? MultiQC is structured to allow easy extension and customisation
              with plugin hooks, a submodule framework and simple templating. Everything is well documented, with step
              by step instructions for writing your new tool.</p>
            <a class="btn btn-default btn-lg" href="docs">Read the docs</a>
          </div>
          <div class="col-md-4 text-center">
            <img class="img-circle img-thumbnail" src="images/documentation.png" alt="MultiQC Documentation" />
          </div>
        </div>
      </section>

      <section>
        <h3>Nice things that people have said about MultiQC</h3>

        <div class="row">
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Amazingly, <a href="https://t.co/QTDfCMVG3B">https://t.co/QTDfCMVG3B</a> just works. Ran `multiqc .` in a dir with bunch of STAR, featurecounts, fastqc results. Worked 1st time.</p>&mdash; Stephen Turner (@genetics_blog) <a href="https://twitter.com/genetics_blog/status/737727882722717696">May 31, 2016</a></blockquote>
          </div>
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-conversation="none" data-lang="en-gb"><p lang="en" dir="ltr"><a href="https://twitter.com/tallphil">@tallphil</a> after using for 1 month, already uncovered discrepancy across samples on a project</p>&mdash; Mike Love (@mikelove) <a href="https://twitter.com/mikelove/status/743778234316296194">17 June 2016</a></blockquote>
          </div>
        </div>
        <div class="row">
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">MultiQC == Awesome tool! Wish all bioinfo software were as documented, modular and useful as this! <a href="https://t.co/OiDAzlNqxS">https://t.co/OiDAzlNqxS</a></p>&mdash; AndrÃ© Rendeiro (@afrendeiro) <a href="https://twitter.com/afrendeiro/status/703325541877882880">February 26, 2016</a></blockquote>
          </div>
          <div class="col-sm-6">
            <div class="reddit-embed" data-embed-media="www.redditmedia.com" data-embed-parent="false" data-embed-live="false" data-embed-uuid="98cc7d46-80e2-428e-a1c4-0b105008ddb4" data-embed-created="2016-06-29T08:50:44.691Z"><a href="https://www.reddit.com/r/bioinformatics/comments/4q8534/multiqc_now_published_tool_for_summarising/d4rjcgh">Comment</a> from discussion <a href="https://www.reddit.com/r/bioinformatics/comments/4q8534/multiqc_now_published_tool_for_summarising/">MultiQC now published! Tool for summarising &amp; visualising bioinfo results.</a>.</div>
          </div>
        </div>
        <div class="row">
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-conversation="none" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/tallphil">@tallphil</a> this such a great difference to the visualization of fastqc reports individually! :)</p>&mdash; Olalla Lorenzo (@o_olalla) <a href="https://twitter.com/o_olalla/status/745212157529296896">June 21, 2016</a></blockquote>
          </div>
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-cards="hidden" data-lang="en"><p lang="en" dir="ltr">Really impressed by this MultiQC tool - Create automatic bioinfo reports: Usage: <a href="https://t.co/EDUmwoxyn1">https://t.co/EDUmwoxyn1</a> Reports: <a href="https://t.co/PsrSUH5Egi">https://t.co/PsrSUH5Egi</a></p>&mdash; Eric Normandeau (@eric_normandeau) <a href="https://twitter.com/eric_normandeau/status/735487628024393728">May 25, 2016</a></blockquote>
          </div>
        </div>
        <div class="row">
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-cards="hidden" data-lang="en"><p lang="en" dir="ltr">Can recommend MultiQC: creates pretty report of -all- output from FastQC,Bowtie,Samtools,etc <a href="https://t.co/cxbV6Nxmq8">https://t.co/cxbV6Nxmq8</a> <a href="https://t.co/4Ha4aupoki">pic.twitter.com/4Ha4aupoki</a></p>&mdash; Elin Videvall (@ElinVidevall) <a href="https://twitter.com/ElinVidevall/status/717753544783360000">April 6, 2016</a></blockquote>
          </div>
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/tallphil">@tallphil</a> MultiQC just blew my mind. Great stuff.</p>&mdash; Stephen Newhouse (@s_j_newhouse) <a href="https://twitter.com/s_j_newhouse/status/755136807986204677">July 18, 2016</a></blockquote>
          </div>
        </div>
        <div class="row">
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-conversation="none" data-lang="en-gb"><p lang="en" dir="ltr"><a href="https://twitter.com/ElinVidevall">@ElinVidevall</a> <a href="https://twitter.com/BioMickWatson">@BioMickWatson</a> <a href="https://twitter.com/tallphil">@tallphil</a> this is a fantastic tool I use frequently for single cell QC. Glad to see it published.</p>&mdash; Sowmya Iyer (@becauseBiology) <a href="https://twitter.com/becauseBiology/status/743783436234686464">17 June 2016</a></blockquote>
          </div>
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-lang="en-gb"><p lang="en" dir="ltr">Current favourite tool - MultiQC - <a href="https://t.co/VsAAS128GJ">https://t.co/VsAAS128GJ</a> - piece of cake to use, essential for large projects, thanks very much <a href="https://twitter.com/tallphil">@tallphil</a></p>&mdash; Lavinia Gordon (@MrsLaviniaG) <a href="https://twitter.com/MrsLaviniaG/status/749099391042981888">2 July 2016</a></blockquote>
          </div>
        </div>
        <div class="row">
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-lang="en-gb"><p lang="en" dir="ltr">I&#39;m continuously impressed by how slick multiqc from <a href="https://twitter.com/tallphil">@tallphil</a> is. It makes my life so much easier.</p>&mdash; Devon Ryan (@dpryan79) <a href="https://twitter.com/dpryan79/status/770579813455257600">30 August 2016</a></blockquote>
          </div>
          <div class="col-sm-6">
            <blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Fantastic tool! Combine 100s of FastQC files and alignment scores into a single report within minutes... <a href="https://t.co/2q0aXkwEl3">https://t.co/2q0aXkwEl3</a></p>&mdash; Jana Sperschneider (@JSperschneider) <a href="https://twitter.com/JSperschneider/status/773337624555565056">September 7, 2016</a></blockquote>
          </div>
        </div>
        <div class="row">
          <div class="col-sm-6">
            <blockquote class="twitter-tweet"><p lang="en" dir="ltr">Tool/Technique Tuesday. Quality controls, sanity checks are very important to ensure one does not enter a never-ending rabbit hole. A great tool that we use all the time is <a href="https://twitter.com/hashtag/MultiQC?src=hash&amp;ref_src=twsrc%5Etfw">#MultiQC</a> from <a href="https://twitter.com/scilifelab?ref_src=twsrc%5Etfw">@scilifelab</a> a tool that scrapes logs for metrics and visualisation. <a href="https://t.co/0nSqslJz1b">https://t.co/0nSqslJz1b</a> <a href="https://t.co/3p6vqJINAj">pic.twitter.com/3p6vqJINAj</a></p>&mdash; MonashBioinformatics (@MonashBioinfo) <a href="https://twitter.com/MonashBioinfo/status/1176278822997524480?ref_src=twsrc%5Etfw">September 23, 2019</a></blockquote>
          </div>
          <div class="col-sm-6">
            <blockquote class="twitter-tweet"><p lang="en" dir="ltr">Just combined about 1200 reports and log files from bowtie2, samtools, samblaster, picard, preseq, cutadapt and fastqc into one human-readable summary report with multiqc in less than 2 minutes. Really awesome! <a href="https://twitter.com/tallphil?ref_src=twsrc%5Etfw">@tallphil</a> <a href="https://t.co/8xagkKeETo">https://t.co/8xagkKeETo</a></p>&mdash; Alexander Toenges (@ATpoint90) <a href="https://twitter.com/ATpoint90/status/1103590478530953219?ref_src=twsrc%5Etfw">March 7, 2019</a></blockquote>
          </div>
        </div>
        <a href="https://twitter.com/search?f=tweets&vertical=default&q=MultiQC" target="_blank" class="btn btn-default">See more on twitter.com &rarr;</a>

        <!-- Twitter & Reddit async scripts -->
        <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
        <script async src="https://www.redditstatic.com/comment-embed.js"></script>

        <!-- <h3>Live discussion about MultiQC on <a href="https://gitter.im/ewels/MultiQC/">Gitter.im</a></h3>
        <iframe src="https://gitter.im/ewels/MultiQC/~embed" style="border:none; width: 100%; height: 300px;"></iframe> -->

      </section>

      <section>
                <ul class="feature-icons">
                    <li><span class="fa fa-lg fa-terminal"></span>Simple installation</li>
                    <li><span class="fa fa-lg fa-search"></span>Search any directory</li>
                    <li><span class="fa fa-lg fa-bar-chart"></span>View report in a web browser</li>
                    <li><span class="fa fa-lg fa-share-alt"></span>Zip file for easy sharing</li>
                    <li><span class="fa fa-lg fa-code"></span>Extensible &amp; well documented</li>
                </ul>
      </section>


      <section class="focus-section">
        <h2>Install from the <a href="https://pypi.python.org/pypi/multiqc">Python Package Index</a> or
        <a href="https://bioconda.github.io/">Bioconda</a></h2>
        <p>To install MultiQC, simply run <code>pip install multiqc</code> on the command line.<br>
        If you use conda, you can run <code>conda install -c bioconda multiqc</code> instead.<br>
        See the <a href="docs/#installation">installation instructions</a> for more help.</p>
        <p>
          <a href="docs" class="btn btn-default btn-lg"><i class="fa fa-book fa-lg"></i> Documentation</a>
          <a target="_blank" href="https://github.com/ewels/MultiQC/" class="btn btn-default btn-lg"><i class="fa fa-github"></i> View on GitHub</a>
          <a target="_blank" href="https://pypi.python.org/pypi/multiqc" class="btn btn-default btn-lg"><i class="icon-python fa-lg"></i> View on PyPI</a>
        </p>
      </section>

    </div></div> <!-- /content /container -->

    <footer id="footer">
      <div class="container">
        <p>Created by Phil Ewels:
          <a target="_blank" href="https://github.com/ewels"><i class="fa fa-github"></i> ewels</a> |
          <a target="_blank" href="https://twitter.com/tallphil"><i class="fa fa-twitter"></i> tallphil</a> |
          <a target="_blank" href="https://se.linkedin.com/in/philewels">LinkedIn</a> |
          <a target="_blank" href="https://www.researchgate.net/profile/Philip_Ewels">ResearchGate</a>
        </p>
        <a target="_blank" href="https://seqera.io/" class="seqera-footer">
          <img src="images/seqera-logo.png">
        </a>
      </div>
    </footer>

    <!-- Supported tools modal -->
    <div class="modal fade" id="supported-tools" tabindex="-1" role="dialog">
      <div class="modal-dialog modal-lg">
        <div class="modal-content">
          <div class="modal-header">
            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
            <h4 class="modal-title">MultiQC: Supported Tools</h4>
          </div>
          <div class="modal-body">
            <div class="alert alert-info">
              MultiQC currently supports <?php echo count($modules); ?> bioinformatics tools, listed below.
              If you would like another to be added, please <a href="https://github.com/ewels/MultiQC/issues" class="alert-link">open an issue</a>.
            </div>
            <ul class="list-inline mod-list-list">
              <li class="mod-list pre-alignment">Pre-alignment tools</li>
              <li class="mod-list aligners">Alignment tools</li>
              <li class="mod-list post-alignment">Post-alignment tools</li>
            </ul>
            <p class="text-muted"><em><span class="fa fa-book"></span> &nbsp;Click the tool name to go to the MultiQC documentation for that tool.</em></p>
            <table class="table table-condensed">
              <tr>
                <th>Tool Name</th>
                <th>Description</th>
              </tr>
              <?php foreach($modules as $mod){
                echo '<tr><td class="mod-list '.strtolower($mod['section']).'"><a href="'.$mod['docs_url'].'">'.$mod['Name'].'</a></td><td><small>'.$mod['Description'].'</small></td></tr>';
              } ?>
            </table>
          </div>
          <div class="modal-footer">
            <button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
          </div>
        </div>
      </div>
    </div>

    <!-- Publication / Citation modal -->
    <div class="modal fade" id="publication" tabindex="-1" role="dialog">
      <div class="modal-dialog modal-lg">
        <div class="modal-content">
          <div class="modal-header">
            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
            <h3 class="modal-title">MultiQC: Publication Details</h3>
          </div>
          <div class="modal-body">
            <div class="alert alert-info">
              Please consider citing MultiQC if you use it in your analysis.
            </div>
            <h4>Citation</h4>
            <blockquote>
              <strong>MultiQC: Summarize analysis results for multiple tools and samples in a single report</strong><br>
              <em>Philip Ewels, M&aring;ns Magnusson, Sverker Lundin and Max K&auml;ller</em><br>
              Bioinformatics (2016)<br>
              doi: <a href="http://dx.doi.org/10.1093/bioinformatics/btw354" target="_blank">10.1093/bioinformatics/btw354</a><br>
              PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/27312411" target="_blank">27312411</a>
            </blockquote>
            <h4>Abstract</h4>
            <div class="well">
              <p><strong>Motivation:</strong> Fast and accurate quality control is essential for studies involving
                next-generation sequencing data.  Whilst numerous tools exist to quantify QC metrics, there is no
                common approach to flexibly integrate these across tools and large sample sets. Assessing analysis
                results across an entire project can be time consuming and error prone; batch effects and outlier
                samples can easily be missed in the early stages of analysis.</p>
              <p><strong>Results:</strong> We present MultiQC, a tool to create a single report visualising output
                from multiple tools across many samples, enabling global trends and biases to be quickly identified.
                MultiQC can plot data from many common bioinformatics tools and is built to allow easy extension
                and customization.</p>
              <p><strong>Availability:</strong> MultiQC is available with an GNU GPLv3 license on GitHub, the Python
                Package Index and Bioconda. Documentation and example reports available at
                <a href="http://multiqc.info">http://multiqc.info</a></p>
            </div>
            <p>This is an Open Access article distributed under the terms of the
              <a href="http://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons Attribution License</a>,
              which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original
              work is properly cited.</p>
            <div class="row" style="margin-top: 40px;">
              <div class="col-sm-6">
                <div data-badge-details="right" data-badge-type="medium-donut" data-doi="10.1093/bioinformatics/btw354" class="altmetric-embed" style="padding-left: 30px;"></div>
              </div>
              <div class="col-sm-6">
                <span class="__dimensions_badge_embed__" data-id="pub.1010049881" data-legend="always"></span>
              </div>
            </div>
          </div>
          <div class="modal-footer">
            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
            <a href="http://dx.doi.org/10.1093/bioinformatics/btw354" class="btn btn-primary">Go to publication</a>
          </div>
        </div>
      </div>
    </div>


    <!-- Placed at the end of the document so the pages load faster -->
    <script src="js/jquery-1.11.3.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <script src="js/homepage.js"></script>

    <!-- Dimensions.ai -->
    <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>

    <!-- Altmetric -->
    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    <!-- Gitter -->
    <script>
      ((window.gitter = {}).chat = {}).options = {
        room: 'ewels/MultiQC',
        activationElement: '#gitter_btn',
      };
    </script>
    <script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>

    <!-- Google Analytics -->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-68098153-1', 'auto');
      ga('send', 'pageview');

    </script>
  </body>
</html>
