<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta http-equiv="Content-Language" content="en">
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="viewport" content="width=device-width" />
    <title>Run Complete - Cluster Flow report</title>
    <meta name="generator" content="Cluster Flow v0.4 devel" />
    <meta name="description" content="Cluster Flow pipeline report: Run Complete" />
    
  </head>
  <body style="width: 100% !important; min-width: 100%; -webkit-text-size-adjust: 100%; -ms-text-size-adjust: 100%; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; text-align: left; line-height: 19px; font-size: 14px; margin: 0; padding: 0;"><style type="text/css">
a:hover {
color: #2795b6 !important;
}
a:active {
color: #2795b6 !important;
}
a:visited {
color: #2ba6cb !important;
}
@media only screen and (min-width: 600px) {
  .header-title-bigscreen {
    display: block;
  }
  .header-title-smallscreen {
    display: none;
  }
}
@media only screen and (max-width: 600px) {
  table[class="body"] img {
    width: auto !important; height: auto !important;
  }
  table[class="body"] center {
    min-width: 0 !important;
  }
  table[class="body"] .container {
    width: 95% !important;
  }
  table[class="body"] .row {
    width: 100% !important; display: block !important;
  }
  table[class="body"] .wrapper {
    display: block !important; padding-right: 0 !important;
  }
  table[class="body"] .columns {
    table-layout: fixed !important; float: none !important; width: 100% !important; padding-right: 0px !important; padding-left: 0px !important; display: block !important;
  }
  table[class="body"] .column {
    table-layout: fixed !important; float: none !important; width: 100% !important; padding-right: 0px !important; padding-left: 0px !important; display: block !important;
  }
  table[class="body"] .wrapper.first .columns {
    display: table !important;
  }
  table[class="body"] .wrapper.first .column {
    display: table !important;
  }
  table[class="body"] table.columns td {
    width: 100% !important;
  }
  table[class="body"] table.column td {
    width: 100% !important;
  }
  table[class="body"] .columns td.one {
    width: 8.333333% !important;
  }
  table[class="body"] .column td.one {
    width: 8.333333% !important;
  }
  table[class="body"] .columns td.two {
    width: 16.666666% !important;
  }
  table[class="body"] .column td.two {
    width: 16.666666% !important;
  }
  table[class="body"] .columns td.three {
    width: 25% !important;
  }
  table[class="body"] .column td.three {
    width: 25% !important;
  }
  table[class="body"] .columns td.four {
    width: 33.333333% !important;
  }
  table[class="body"] .column td.four {
    width: 33.333333% !important;
  }
  table[class="body"] .columns td.five {
    width: 41.666666% !important;
  }
  table[class="body"] .column td.five {
    width: 41.666666% !important;
  }
  table[class="body"] .columns td.six {
    width: 50% !important;
  }
  table[class="body"] .column td.six {
    width: 50% !important;
  }
  table[class="body"] .columns td.seven {
    width: 58.333333% !important;
  }
  table[class="body"] .column td.seven {
    width: 58.333333% !important;
  }
  table[class="body"] .columns td.eight {
    width: 66.666666% !important;
  }
  table[class="body"] .column td.eight {
    width: 66.666666% !important;
  }
  table[class="body"] .columns td.nine {
    width: 75% !important;
  }
  table[class="body"] .column td.nine {
    width: 75% !important;
  }
  table[class="body"] .columns td.ten {
    width: 83.333333% !important;
  }
  table[class="body"] .column td.ten {
    width: 83.333333% !important;
  }
  table[class="body"] .columns td.eleven {
    width: 91.666666% !important;
  }
  table[class="body"] .column td.eleven {
    width: 91.666666% !important;
  }
  table[class="body"] .columns td.twelve {
    width: 100% !important;
  }
  table[class="body"] .column td.twelve {
    width: 100% !important;
  }
  table[class="body"] table.columns td.expander {
    width: 1px !important;
  }
  .header-title-bigscreen {
    display: none;
  }
  .header-title-smallscreen {
    display: block;
  }
}
</style>
	<table class="body" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; height: 100%; width: 100%; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0;"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td class="center" align="center" valign="top" style="border-collapse: collapse !important; vertical-align: top; text-align: center; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0;">
        <center style="width: 100%; min-width: 580px;">

          <table class="row header" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; width: 100%; position: relative; background: #2d343c; padding: 0px;" bgcolor="#2d343c"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td class="center" align="center" style="border-collapse: collapse !important; vertical-align: top; text-align: center; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0;" valign="top">
                <center style="width: 100%; min-width: 580px;">

                  <table class="container" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: inherit; width: 580px; margin: 0 auto; padding: 0;"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td class="wrapper last" style="border-collapse: collapse !important; vertical-align: top; text-align: left; position: relative; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 10px 0px 0px;" align="left" valign="top">

                        <table class="twelve columns title-header" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; width: 580px; margin: 0 auto; padding: 0;"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td class="six sub-columns" style="border-collapse: collapse !important; vertical-align: top; text-align: left; min-width: 0px; width: 50%; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0px 10px 10px 0px;" align="left" valign="top">
                              <img title="Cluster Flow" src="http://clusterflow.io/img/CF_email_logo.png" style="outline: none; text-decoration: none; -ms-interpolation-mode: bicubic; width: 232px; max-width: 100%; float: left; clear: both; display: block; height: 74px; color: #FFFFFF; font-size: 25px; padding-left:10px;" align="left" /></td>
                            <td class="six sub-columns last" style="text-align: right; vertical-align: middle; border-collapse: collapse !important; min-width: 0px; width: 50%; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0px 0px 10px;" align="right" valign="middle">
                              <h1 class="header-title-bigscreen" style="color: #FFFFFF; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; text-align: right; line-height: 1.3; word-break: normal; font-size: 40px; margin: 0; padding: 0 10px 0 0;" align="right">Run Complete</h1>
                            </td>
                            <td class="expander" style="border-collapse: collapse !important; vertical-align: top; text-align: left; visibility: hidden; width: 0px; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0;" align="left" valign="top"></td>
                          </tr></table></td>
                    </tr></table></center>
              </td>
            </tr></table><table class="container" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: inherit; width: 580px; margin: 0 auto; padding: 0;"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td style="border-collapse: collapse !important; vertical-align: top; text-align: left; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0;" align="left" valign="top">

                <table class="row" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; width: 100%; position: relative; display: block; padding: 0px;"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td class="wrapper last" style="border-collapse: collapse !important; vertical-align: top; text-align: left; position: relative; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 10px 0px 0px;" align="left" valign="top">

                      <table class="twelve columns" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; width: 580px; margin: 0 auto; padding: 0;"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td style="border-collapse: collapse !important; vertical-align: top; text-align: left; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0px 0px 10px;" align="left" valign="top">
														
														<h1 class="header-title-smallscreen" style="color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; text-align: left; line-height: 1.3; word-break: normal; font-size: 40px; margin: 0 0 10px; padding: 0; display:none;" align="left">Run Complete</h1>
														<p class="completion-leader" style="color: #723736; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; text-align: center; line-height: 19px; font-size: 18px; border-radius: 5px; background: #f2dede; margin: 0 0 15px; padding: 10px; border: 1px solid #ebccd1;" align="center">
The pipeline <strong>sra_star</strong> has completed <strong>with errors</strong></p>

<hr style="color: #d9d9d9; height: 1px; background: #d9d9d9; border: none;" />

<table class="run-stats" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; padding: 0;">
	<tr style="vertical-align: top; text-align: left; padding: 0;" align="left">
		<th style="text-align: right; padding-right: 10px; min-width: 70px;" align="right">
			Started
		</th>
		<td style="border-collapse: collapse !important; vertical-align: top; text-align: left; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0px 0px 10px;" align="left" valign="top">
			17:30 02-05-2016
		</td>
	</tr>
	<tr>
		<th style="text-align: right; padding-right: 10px; min-width: 70px;" align="right">
			Finished
		</th>
		<td style="border-collapse: collapse !important; vertical-align: top; text-align: left; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0px 0px 10px;" align="left" valign="top">
			07:20 03-05-2016
		</td>
	</tr>
	<tr>
		<th style="text-align: right; padding-right: 10px; min-width: 70px;" align="right">
			Duration
		</th>
		<td style="border-collapse: collapse !important; vertical-align: top; text-align: left; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0px 0px 10px;" align="left" valign="top">
			13 hours, 50 minutes, 15 seconds
		</td>
	</tr>
	<tr>
		<th style="text-align: right; padding-right: 10px; min-width: 70px;" align="right">
			Working directory
		</th>
		<td style="border-collapse: collapse !important; vertical-align: top; text-align: left; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0px 0px 10px;" align="left" valign="top">
			<code style="font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">
				/pica/v12/b2013064_nobackup/private/phil/demo_RNAseq/
			</code>
		</td>
	</tr>
</table>

<hr style="color: #d9d9d9; height: 1px; background: #d9d9d9; border: none;" />



<h3 style="color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; text-align: left; line-height: 1.3; word-break: normal; font-size: 32px; margin: 0; padding: 0;" align="left">CF Status Messages</h3>
<ul class="status-messages" style="padding-left:20px;"><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192396.sra_sra_star_clusterFlow.txt</span><ul style="padding-left:20px;"><li style="margin-top: 5px;">Download worked - took 14 minutes, 27 seconds</li><li style="margin-top: 5px;">FastQ Dump successfully exited on attempt 1, took 20 minutes, 6 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 15 minutes, 47 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 15 minutes, 4 seconds</li><li style="margin-top: 5px;">Trim galore (PE mode) successfully exited, took 3 hours, 19 minutes, 41 seconds..</li><li style="margin-top: 5px;">STAR (PE mode) successfully exited, took 1 hours, 29 minutes, 33 seconds..</li><li style="margin-top: 5px;">featureCounts successfully exited, took 10 minutes, 41 seconds</li><li style="margin-top: 5px;">samtools sort successfully exited, took 1 hours, 12 minutes, 47 seconds..</li><li style="margin-top: 5px;">samtools index successfully exited, took 3 minutes, 45 seconds.</li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! Preseq (PE mode) exited in an error state for input file 'SRR3192396_1_star_aligned_srtd.bam': 256 </li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! No file names found from job cf_sra_star_1462203013_preseq_calc_401. Exiting...</li><li style="margin-top: 5px;">Run finished at 07:15 03-05-2016</li></ul></li><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192397.sra_sra_star_clusterFlow.txt</span><ul style="padding-left:20px;"><li style="margin-top: 5px;">Download worked - took 12 minutes, 14 seconds</li><li style="margin-top: 5px;">FastQ Dump successfully exited on attempt 1, took 15 minutes, 39 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 12 minutes, 52 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 13 minutes, 29 seconds</li><li style="margin-top: 5px;">Trim galore (PE mode) successfully exited, took 3 hours, 7 minutes, 30 seconds..</li><li style="margin-top: 5px;">STAR (PE mode) successfully exited, took 1 hours, 9 minutes, 31 seconds..</li><li style="margin-top: 5px;">featureCounts successfully exited, took 9 minutes, 27 seconds</li><li style="margin-top: 5px;">samtools sort successfully exited, took 1 hours, 4 minutes, 34 seconds..</li><li style="margin-top: 5px;">samtools index successfully exited, took 3 minutes, 5 seconds.</li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! Preseq (PE mode) exited in an error state for input file 'SRR3192397_1_star_aligned_srtd.bam': 256 </li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! No file names found from job cf_sra_star_1462203013_preseq_calc_688. Exiting...</li><li style="margin-top: 5px;">Run finished at 04:22 03-05-2016</li></ul></li><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192398.sra_sra_star_clusterFlow.txt</span><ul style="padding-left:20px;"><li style="margin-top: 5px;">Download worked - took 13 minutes, 7 seconds</li><li style="margin-top: 5px;">FastQ Dump successfully exited on attempt 1, took 11 minutes, 2 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 9 minutes, 36 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 9 minutes, 13 seconds</li><li style="margin-top: 5px;">Trim galore (PE mode) successfully exited, took 2 hours, 21 minutes, 1 seconds..</li><li style="margin-top: 5px;">STAR (PE mode) successfully exited, took 47 minutes, 25 seconds..</li><li style="margin-top: 5px;">featureCounts successfully exited, took 6 minutes, 36 seconds</li><li style="margin-top: 5px;">samtools sort successfully exited, took 53 minutes, 45 seconds..</li><li style="margin-top: 5px;">samtools index successfully exited, took 2 minutes, 21 seconds.</li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! Preseq (PE mode) exited in an error state for input file 'SRR3192398_1_star_aligned_srtd.bam': 256 </li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! No file names found from job cf_sra_star_1462203013_preseq_calc_415. Exiting...</li><li style="margin-top: 5px;">Run finished at 05:14 03-05-2016</li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error: ngi_visualizations Python Package not installed.</li><li style="margin-top: 5px;">Pipeline sra_star finished at 07:20 03-05-2016</li></ul></li><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192399.sra_sra_star_clusterFlow.txt</span><ul style="padding-left:20px;"><li style="margin-top: 5px;">Download worked - took 9 minutes, 25 seconds</li><li style="margin-top: 5px;">FastQ Dump successfully exited on attempt 1, took 12 minutes, 25 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 9 minutes, 56 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 9 minutes, 56 seconds</li><li style="margin-top: 5px;">Trim galore (PE mode) successfully exited, took 2 hours, 33 minutes, 17 seconds..</li><li style="margin-top: 5px;">STAR (PE mode) successfully exited, took 53 minutes, 35 seconds..</li><li style="margin-top: 5px;">featureCounts successfully exited, took 8 minutes, 7 seconds</li><li style="margin-top: 5px;">samtools sort successfully exited, took 55 minutes, 41 seconds..</li><li style="margin-top: 5px;">samtools index successfully exited, took 3 minutes, 46 seconds.</li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! Preseq (PE mode) exited in an error state for input file 'SRR3192399_1_star_aligned_srtd.bam': 256 </li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! No file names found from job cf_sra_star_1462203013_preseq_calc_566. Exiting...</li><li style="margin-top: 5px;">Run finished at 05:12 03-05-2016</li></ul></li><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192400.sra_sra_star_clusterFlow.txt</span><ul style="padding-left:20px;"><li style="margin-top: 5px;">Download worked - took 14 minutes, 55 seconds</li><li style="margin-top: 5px;">FastQ Dump successfully exited on attempt 1, took 15 minutes, 48 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 12 minutes, 34 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 12 minutes, 50 seconds</li><li style="margin-top: 5px;">Trim galore (PE mode) successfully exited, took 3 hours, 8 minutes, 18 seconds..</li><li style="margin-top: 5px;">STAR (PE mode) successfully exited, took 1 hours, 9 minutes, 46 seconds..</li><li style="margin-top: 5px;">featureCounts successfully exited, took 9 minutes, 3 seconds</li><li style="margin-top: 5px;">samtools sort successfully exited, took 56 minutes, 9 seconds..</li><li style="margin-top: 5px;">samtools index successfully exited, took 2 minutes, 53 seconds.</li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! Preseq (PE mode) exited in an error state for input file 'SRR3192400_1_star_aligned_srtd.bam': 256 </li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! No file names found from job cf_sra_star_1462203013_preseq_calc_153. Exiting...</li><li style="margin-top: 5px;">Run finished at 05:32 03-05-2016</li></ul></li><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192401.sra_sra_star_clusterFlow.txt</span><ul style="padding-left:20px;"><li style="margin-top: 5px;">Download worked - took 27 minutes, 22 seconds</li><li style="margin-top: 5px;">FastQ Dump successfully exited on attempt 1, took 15 minutes, 17 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 12 minutes, 49 seconds</li><li style="margin-top: 5px;">FastQC successfully ran, took 13 minutes, 5 seconds</li><li style="margin-top: 5px;">Trim galore (PE mode) successfully exited, took 3 hours, 15 minutes, 21 seconds..</li><li style="margin-top: 5px;">STAR (PE mode) successfully exited, took 1 hours, 16 minutes, 39 seconds..</li><li style="margin-top: 5px;">featureCounts successfully exited, took 9 minutes, 38 seconds</li><li style="margin-top: 5px;">samtools sort successfully exited, took 1 hours, 1 minutes, 12 seconds..</li><li style="margin-top: 5px;">samtools index successfully exited, took 2 minutes, 55 seconds.</li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! Preseq (PE mode) exited in an error state for input file 'SRR3192401_1_star_aligned_srtd.bam': 256 </li><li style="margin-top: 5px; background: #ebccd1; color: #222222; font-weight:bold; padding: 3px 5px;">Error! No file names found from job cf_sra_star_1462203013_preseq_calc_876. Exiting...</li><li style="margin-top: 5px;">Run finished at 05:35 03-05-2016</li></ul></li>
</ul>

<hr style="color: #d9d9d9; height: 1px; background: #d9d9d9; border: none;" />
<h3 style="color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; text-align: left; line-height: 1.3; word-break: normal; font-size: 32px; margin: 0; padding: 0;" align="left">Commands Run</h3>

<ul class="status-messages commands-run" style="padding:0; list-style-type:none;"><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192396.sra_sra_star_clusterFlow.txt</span><ul style="padding:0; list-style-type:none;"><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">wget -nv --tries=10 --output-document=SRR3192396.sra ftp://ftp-trace.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByRun/sra/SRR/SRR319/SRR3192396/SRR3192396.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastq-dump --split-files ./SRR3192396.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192396_1.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192396_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">trim_galore --paired --gzip --phred33       --fastqc SRR3192396_1.fastq.gz SRR3192396_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">STAR --runThreadN 4 --outQSconversionAdd 0 --outSAMattributes Standard --genomeLoad NoSharedMemory  --readFilesCommand zcat --genomeDir /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Sequence/STARIndex/ --readFilesIn SRR3192396_1_val_1.fq.gz SRR3192396_2_val_2.fq.gz --outFileNamePrefix SRR3192396_1 --outStd SAM | samtools view -bS - > SRR3192396_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">featureCounts -T 1 -p  -t exon -g gene_id -a /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Annotation/Genes/genes.gtf -o SRR3192396_1_star_aligned.bam_counts.txt SRR3192396_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools sort -m 4G  SRR3192396_1_star_aligned.bam SRR3192396_1_star_aligned_srtd</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools index SRR3192396_1_star_aligned_srtd.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">preseq lc_extrap -e 1000000000 -v -Q -B -P -l 999999999999 SRR3192396_1_star_aligned_srtd.bam -o SRR3192396_1_star_aligned_srtd.bam.preseq</li></ul></li><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192397.sra_sra_star_clusterFlow.txt</span><ul style="padding:0; list-style-type:none;"><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">wget -nv --tries=10 --output-document=SRR3192397.sra ftp://ftp-trace.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByRun/sra/SRR/SRR319/SRR3192397/SRR3192397.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastq-dump --split-files ./SRR3192397.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192397_1.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192397_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">trim_galore --paired --gzip --phred33       --fastqc SRR3192397_1.fastq.gz SRR3192397_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">STAR --runThreadN 4 --outQSconversionAdd 0 --outSAMattributes Standard --genomeLoad NoSharedMemory  --readFilesCommand zcat --genomeDir /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Sequence/STARIndex/ --readFilesIn SRR3192397_1_val_1.fq.gz SRR3192397_2_val_2.fq.gz --outFileNamePrefix SRR3192397_1 --outStd SAM | samtools view -bS - > SRR3192397_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">featureCounts -T 1 -p  -t exon -g gene_id -a /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Annotation/Genes/genes.gtf -o SRR3192397_1_star_aligned.bam_counts.txt SRR3192397_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools sort -m 4G  SRR3192397_1_star_aligned.bam SRR3192397_1_star_aligned_srtd</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools index SRR3192397_1_star_aligned_srtd.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">preseq lc_extrap -e 1000000000 -v -Q -B -P -l 999999999999 SRR3192397_1_star_aligned_srtd.bam -o SRR3192397_1_star_aligned_srtd.bam.preseq</li></ul></li><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192398.sra_sra_star_clusterFlow.txt</span><ul style="padding:0; list-style-type:none;"><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">wget -nv --tries=10 --output-document=SRR3192398.sra ftp://ftp-trace.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByRun/sra/SRR/SRR319/SRR3192398/SRR3192398.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastq-dump --split-files ./SRR3192398.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192398_1.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192398_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">trim_galore --paired --gzip --phred33       --fastqc SRR3192398_1.fastq.gz SRR3192398_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">STAR --runThreadN 4 --outQSconversionAdd 0 --outSAMattributes Standard --genomeLoad NoSharedMemory  --readFilesCommand zcat --genomeDir /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Sequence/STARIndex/ --readFilesIn SRR3192398_1_val_1.fq.gz SRR3192398_2_val_2.fq.gz --outFileNamePrefix SRR3192398_1 --outStd SAM | samtools view -bS - > SRR3192398_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">featureCounts -T 1 -p  -t exon -g gene_id -a /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Annotation/Genes/genes.gtf -o SRR3192398_1_star_aligned.bam_counts.txt SRR3192398_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools sort -m 4G  SRR3192398_1_star_aligned.bam SRR3192398_1_star_aligned_srtd</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools index SRR3192398_1_star_aligned_srtd.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">preseq lc_extrap -e 1000000000 -v -Q -B -P -l 999999999999 SRR3192398_1_star_aligned_srtd.bam -o SRR3192398_1_star_aligned_srtd.bam.preseq</li></ul></li><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192399.sra_sra_star_clusterFlow.txt</span><ul style="padding:0; list-style-type:none;"><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">wget -nv --tries=10 --output-document=SRR3192399.sra ftp://ftp-trace.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByRun/sra/SRR/SRR319/SRR3192399/SRR3192399.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastq-dump --split-files ./SRR3192399.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192399_1.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192399_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">trim_galore --paired --gzip --phred33       --fastqc SRR3192399_1.fastq.gz SRR3192399_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">STAR --runThreadN 4 --outQSconversionAdd 0 --outSAMattributes Standard --genomeLoad NoSharedMemory  --readFilesCommand zcat --genomeDir /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Sequence/STARIndex/ --readFilesIn SRR3192399_1_val_1.fq.gz SRR3192399_2_val_2.fq.gz --outFileNamePrefix SRR3192399_1 --outStd SAM | samtools view -bS - > SRR3192399_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">featureCounts -T 1 -p  -t exon -g gene_id -a /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Annotation/Genes/genes.gtf -o SRR3192399_1_star_aligned.bam_counts.txt SRR3192399_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools sort -m 4G  SRR3192399_1_star_aligned.bam SRR3192399_1_star_aligned_srtd</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools index SRR3192399_1_star_aligned_srtd.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">preseq lc_extrap -e 1000000000 -v -Q -B -P -l 999999999999 SRR3192399_1_star_aligned_srtd.bam -o SRR3192399_1_star_aligned_srtd.bam.preseq</li></ul></li><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192400.sra_sra_star_clusterFlow.txt</span><ul style="padding:0; list-style-type:none;"><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">wget -nv --tries=10 --output-document=SRR3192400.sra ftp://ftp-trace.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByRun/sra/SRR/SRR319/SRR3192400/SRR3192400.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastq-dump --split-files ./SRR3192400.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192400_1.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192400_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">trim_galore --paired --gzip --phred33       --fastqc SRR3192400_1.fastq.gz SRR3192400_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">STAR --runThreadN 4 --outQSconversionAdd 0 --outSAMattributes Standard --genomeLoad NoSharedMemory  --readFilesCommand zcat --genomeDir /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Sequence/STARIndex/ --readFilesIn SRR3192400_1_val_1.fq.gz SRR3192400_2_val_2.fq.gz --outFileNamePrefix SRR3192400_1 --outStd SAM | samtools view -bS - > SRR3192400_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">featureCounts -T 1 -p  -t exon -g gene_id -a /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Annotation/Genes/genes.gtf -o SRR3192400_1_star_aligned.bam_counts.txt SRR3192400_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools sort -m 4G  SRR3192400_1_star_aligned.bam SRR3192400_1_star_aligned_srtd</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools index SRR3192400_1_star_aligned_srtd.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">preseq lc_extrap -e 1000000000 -v -Q -B -P -l 999999999999 SRR3192400_1_star_aligned_srtd.bam -o SRR3192400_1_star_aligned_srtd.bam.preseq</li></ul></li><li style="margin-bottom: 20px;">
	<span class="run-name" style="font-weight: bold; margin-bottom: 10px;">
	SRR3192401.sra_sra_star_clusterFlow.txt</span><ul style="padding:0; list-style-type:none;"><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">wget -nv --tries=10 --output-document=SRR3192401.sra ftp://ftp-trace.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByRun/sra/SRR/SRR319/SRR3192401/SRR3192401.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastq-dump --split-files ./SRR3192401.sra</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192401_1.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">fastqc -q  SRR3192401_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">trim_galore --paired --gzip --phred33       --fastqc SRR3192401_1.fastq.gz SRR3192401_2.fastq.gz</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">STAR --runThreadN 4 --outQSconversionAdd 0 --outSAMattributes Standard --genomeLoad NoSharedMemory  --readFilesCommand zcat --genomeDir /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Sequence/STARIndex/ --readFilesIn SRR3192401_1_val_1.fq.gz SRR3192401_2_val_2.fq.gz --outFileNamePrefix SRR3192401_1 --outStd SAM | samtools view -bS - > SRR3192401_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">featureCounts -T 1 -p  -t exon -g gene_id -a /sw/data/uppnex/igenomes/Homo_sapiens/Ensembl/GRCh37/Annotation/Genes/genes.gtf -o SRR3192401_1_star_aligned.bam_counts.txt SRR3192401_1_star_aligned.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools sort -m 4G  SRR3192401_1_star_aligned.bam SRR3192401_1_star_aligned_srtd</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">samtools index SRR3192401_1_star_aligned_srtd.bam</li><li style="margin-top: 5px; font-family: 'Lucida Console', Monaco, monospace; font-size: 12px; background: #efefef; padding: 3px 5px;">preseq lc_extrap -e 1000000000 -v -Q -B -P -l 999999999999 SRR3192401_1_star_aligned_srtd.bam -o SRR3192401_1_star_aligned_srtd.bam.preseq</li></ul></li></ul><hr style="color: #d9d9d9; height: 1px; background: #d9d9d9; border: none;" />
														
														</td>
                          <td class="expander" style="border-collapse: collapse !important; vertical-align: top; text-align: left; visibility: hidden; width: 0px; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0;" align="left" valign="top"></td>
                        </tr></table></td>
                  </tr></table>
									<table class="row" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; width: 100%; position: relative; display: block; padding: 0px;"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td class="wrapper last" style="border-collapse: collapse !important; vertical-align: top; text-align: left; position: relative; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 10px 0px 0px;" align="left" valign="top">

                      <table class="twelve columns" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; width: 580px; margin: 0 auto; padding: 0;"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td align="center" style="border-collapse: collapse !important; vertical-align: top; text-align: left; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0px 0px 10px;" valign="top">
                            <center style="width: 100%; min-width: 580px;">
															<p style="text-align: center; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0 0 10px; padding: 0;" align="center">Generated by <a href="http://clusterflow.io" style="color: #2ba6cb; text-decoration: none;">Cluster Flow</a> v0.4 devel</p>
                              <p style="text-align: center; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0 0 10px; padding: 0;" align="center"><a href="http://clusterflow.io" style="color: #2ba6cb; text-decoration: none;">Documentation</a> | <a href="http://clusterflow.io/0.4/reference" style="color: #2ba6cb; text-decoration: none;">Notifications</a></p>
                            </center>
                          </td>
                          <td class="expander" style="border-collapse: collapse !important; vertical-align: top; text-align: left; visibility: hidden; width: 0px; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0;" align="left" valign="top"></td>
                        </tr></table></td>
                  </tr></table><!-- container end below --></td>
            </tr></table></center>
			</td>
		</tr></table></body>
</html>
